### PR TITLE
Add mongooseimctl bootstrap command

### PR DESCRIPTION
**Pull request by arcusfelis**
Proposed changes include:
* ejabberdctl command and docs

### CODE REVIEW

**Comment by mongoose-im** [comment]
[8070.1](https://travis-ci.org/esl/MongooseIM/jobs/668694067) / Erlang 22.0 / small_tests / 2e348f44c211a8dd4cc572891831e9539a92952b
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2692/8070/small_tests.22.0) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2692/8070/small_tests.22.0/small/ct_run.mongooseim@localhost.2020-03-30_11.00.30/lib.mongooseim.logs/run.2020-03-30_11.00.31/suite.log.html)

File: doc/user-guide/Bootstrap-Scripts.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Bootstrap scripts
+
+Scripts are located in `rel/files/scripts/` directory in the MongooseIM repository.
```
Comment (goddammit) [inline doc/user-guide/Bootstrap-Scripts.md]:
```suggestion
The scripts are located in the `rel/files/scripts/` directory in the MongooseIM repository.
```

File: doc/user-guide/Bootstrap-Scripts.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+By default `bootstrap` command executes `bootstrap01-hello.sh`, which just prints the information below:
```
Comment (goddammit) [inline doc/user-guide/Bootstrap-Scripts.md]:
```suggestion
By default the `bootstrap` command executes `bootstrap01-hello.sh`, which just prints the information below:
```

File: doc/user-guide/Getting-started.md [hunk 3]
```diff
@@ -81,6 +81,18 @@ You can check server loglevel:
 mongooseimctl get_loglevel
 ```
 
+Run bootstrap scripts for initial configuration:
+
+```
+mongooseimctl bootstrap
+```
+
+It executes scripts inside `scripts/` directory with `bootstrap` prefix in alphabetical order. [More information](Bootstrap-Scripts.md)
```
Comment (goddammit) [inline doc/user-guide/Getting-started.md]:
```suggestion
It executes scripts inside the `scripts/` directory with a `bootstrap` prefix in alphabetical order. [More information](Bootstrap-Scripts.md)
```

File: rel/files/mongooseimctl [hunk 4]
```diff
@@ -11,14 +11,15 @@ if [ ! -z "$RUNNER_USER" ] && [ `whoami` != "$RUNNER_USER" ]; then
     exec runuser -l "$RUNNER_USER" -c "$0 $*"
 fi
 
-EJABBERD_DIR="${RUNNER_SCRIPT_DIR%/*}"
+MIM_DIR="${RUNNER_SCRIPT_DIR%/*}"
```
Comment (michalwski) [inline rel/files/mongooseimctl]: Thanks a lot for the name change!

File: doc/user-guide/Bootstrap-Scripts.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Bootstrap scripts
```
Comment (michalwski) [inline doc/user-guide/Bootstrap-Scripts.md]: I think it'd be worth adding a few words how the `bootstrap` command behaves when a script inside the `scripts` directory returns with an error. Do other scripts are run, or the execution is terminated? 
What if there will be a text file with correct prefix?

**Comment by mongoose-im** [comment]
[8111.1](https://travis-ci.org/esl/MongooseIM/jobs/671638109) / Erlang 22.0 / small_tests / ef6fd4e6ff367ccc6da8488c70893729af584ada
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2692/8111/small_tests.22.0) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2692/8111/small_tests.22.0/small/ct_run.mongooseim@localhost.2020-04-06_13.35.35/lib.mongooseim.logs/run.2020-04-06_13.35.35/suite.log.html)

**Pull request merged**